# Message for reporting the height and angle of the camera
float64 height      # height of camera from ground
float64 angle       # angle of camera down from horizontal
